src/patternfly/components/Page/page.scss

```diff
@@ -202,6 +206,17 @@ $pf-c-page--breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg", "xl",
   @media screen and (min-width: $pf-global--breakpoint--lg) {
     grid-template-columns: auto 1fr auto;
   }
+
+  .pf-c-masthead {
+    --pf-c-masthead--inset: var(--pf-c-page__header--c-masthead--inset);
```

Comment:
Oh I see what you did here in setting the insets. I would have set the page insets as default, or just used `.pf-m-page-insets` instead of setting the vars here, too. I would have set the toggle button font-size below in the masthead component, too, as the default. I'm curious what the use cases for the masthead are outside of our page or why you would want to change insets or font size?